esp/esp/program/modules/base.py

```diff
@@ -389,9 +365,6 @@ def isCompleted(self):
     def prepare(self, context):
         return context
 
-    def getNavBars(self):
```

Comment:
Yeah, looks like it was intended to automatically add entries to navbars when the module was enabled? That's not even something we want anymore.